### PR TITLE
Style Welcome Box

### PR DESCRIPTION
**Pull request by pradeepgangwar**
Associated Issue: #3830 

### Summary of Changes

* change style of command and info text
* add a new shortcut
* remove "+" sign from commands

### Screenshots

* Currently It looks like this. So I need further help.

screenshot-2017-8-31 debugger

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@codehag Thanks. But There were few more changes to be implemented according to issue but i am stuck. One of them is removing "+" sign from "Shift+". Can anybody help me?

File: src/utils/text.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ function formatKeyShortcut(shortcut: string): string {
   }
   return shortcut.replace(
     /CommandOrControl\+|CmdOrCtrl\+/g,
```
Comment (wldcordeiro) [inline src/utils/text.js]: I think if you want to remove the `+` you just remove the two `\+` in this line.

**Comment by pradeepgangwar** [comment]
@wldcordeiro I was able to achieve that by adding line `.replace(/Shift\+/g, "Shift ");` below the line you mentioned. 

But Issue also says to add cmd+shift+o as shortcut. So can somebody explain me how Line no. 42 and 46 of welcomebox.js work. Because that is what make key combination.

**Comment by nyrosmith** [comment]
Instead of replace usw .split("+").join(" ")

**Comment by pradeepgangwar** [comment]
can somebody explain how 
`const keyCombinationTwo = formatKeyShortcut(    L10N.getStr("projectTextSearch.key") );`
 in welcomebox.js [Link here](https://github.com/devtools-html/debugger.html/blob/master/src/components/WelcomeBox.js#L42) creates a new combination. Because according to issue i have to make a new shortcut key combination.

**Comment by jasonLaster** [comment]
@pradeepgangwar we might want two different functions...

**Comment by pradeepgangwar** [comment]
newone

@jasonLaster  This is final thing I did from my side. Please review it and suggest edits if anything is wrong.

File: assets/panel/debugger.properties [hunk 2]
```diff
@@ -440,6 +444,11 @@ welcome.search=%S to search for sources
 # a mac we use the unicode character.
 welcome.findInFiles=%S to find in files
 
+# LOCALIZATION NOTE (welcome.searchFunction): The center pane welcome panel's
```
Comment (flodolo) [inline assets/panel/debugger.properties]: Nit: simply explain what's going to replace the placeholder.

```
# LOCALIZATION NOTE (welcome.searchFunction): Label displayed in the welcome
# panel's. %S is replaced by the keyboard shortcut to search for functions.
```

File: assets/panel/debugger.properties [hunk 3]
```diff
@@ -440,6 +444,11 @@ welcome.search=%S to search for sources
...
+# search prompt. e.g. cmd+o to search for files. On windows, it's ctrl+shift+o, on
+# a mac we use the unicode character.
+welcome.searchFunction=%S to search for function in file
```
Comment (flodolo) [inline assets/panel/debugger.properties]: I think this should either be "for a function" or "for functions"? (not a native English speaker though)

**Comment by pradeepgangwar** [comment]
@flodolo made those changes. Thanks.

File: assets/panel/debugger.properties [hunk 4]
```diff
@@ -440,6 +444,10 @@ welcome.search=%S to search for sources
 # a mac we use the unicode character.
 welcome.findInFiles=%S to find in files
 
+# LOCALIZATION NOTE (welcome.searchFunction): Label displayed in the welcome
+# panel's. %S is replaced by the keyboard shortcut to search for functions.
```
Comment (flodolo) [inline assets/panel/debugger.properties]: Ugh, the genitive after panel should not be there (that's my mistake).

**Comment by pradeepgangwar** [comment]
@flodolo Done typo fixes. Thanks for suggestions.

**Comment by jasonLaster** [comment]
This is looking great :)

Couple UI nits:

1. line up the left hand shortkeys line up
2. reduce the space between shortcuts and descriptions
3. i think the shortcuts are bolded can we try lightening it

**Comment by jasonLaster** [comment]
Thanks @pradeepgangwar!

**Pull request merged**